src/loaders/transactions/disease_gene.py

```diff
@@ -10,68 +10,66 @@ def disease_gene_object_tx(self, data):
         # Loads the Disease data into Neo4j.
 
         executeGene = """
-
+        
             UNWIND $data as row
-
+            
             // GET PRIMARY DATA OBJECTS
-
             MATCH (d:DOTerm:Ontology {primaryKey:row.doId})
             MATCH (gene:Gene {primaryKey:row.primaryId})
-
+            
             // LOAD NODES
-
-            MERGE (l:Load {primaryKey:row.loadKey})
+            MERGE (l:Load:Entity {primaryKey:row.loadKey})
                 SET l.dateProduced = row.dateProduced
-                SET l.dataProvider = row.dataProvider
                 SET l.loadName = "Disease"
-
-            MERGE (spec:Species {primaryKey: row.taxonId})
-            MERGE (gene)<-[:FROM_SPECIES]->(spec)
-
+                SET l.dataProviders = row.dataProviders
+                SET l.dataProvider = row.dataProvider
+                
              MERGE (dga:Association {primaryKey:row.uuid})  
                 SET dga :DiseaseEntityJoin
-
+                SET dga.dataProviders = row.dataProviders
+                SET dga.dataProvider = row.dataProvider
+                
+            MERGE (dfa)-[dfal:LOADED_FROM]-(l)
+                
             FOREACH (rel IN CASE when row.relationshipType = 'is_marker_for' THEN [1] ELSE [] END | 
                 MERGE (gene)<-[fafg:IS_MARKER_FOR {uuid:row.uuid}]->(d) 
```

Comment:
Any idea about these weird characters? I see them on a few lines below.

Reply:
@christabone - sorry which weird characters?  I'm not seeing them...

Reply:
They're a bit strange -- I took a screenshot:

![image](https://user-images.githubusercontent.com/3584662/41131480-fe61167e-6a89-11e8-80d4-52b7f2bac365.png)